hieradata/hosts/test101.yaml

```diff
@@ -10,8 +10,7 @@ php::php_version: '7.4'
 mediawiki::php::use_tideways: true
 nginx::use_graylog: true
 puppetserver_hostname: 'puppet111.miraheze.org'
-mediawiki::php::fpm::childs: 12
-mediawiki::php::fpm::fpm_max_memory: 256
+mediawiki::php::fpm::childs: 32
```

Comment:
Shouldn't have 32 childs. It doesn't have the same ram as the mw*. Same fro test3

Reply:
The PR just fully reverts them to what they were prior.